config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  root 'root#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] Kindly deleted this commented line of code and any other commented lines of code in your codebase. We have to always keep our codebase clean and professional